Documentation/ColumnsConfig/Type/Flex/FlexformSyntax.rst

```diff
@@ -27,7 +27,14 @@ documented here for TCA. The limitations are:
 -  Charset follows that of the current backend UTF-8. When storing FlexForm information in external files,
    make sure that they are using UTF-8 too.
 
--  :php:`type='inline'` and other type's that point to different tables are not allowed in flex form section containers.
+-  :php:`type='inline'` and other types that point to different tables are not allowed in flex form section containers.
+
+.. versionchanged:: 13.0
+
+    Since TYPO3 13.0, also :php:`type='select'` (when using
+    :php`foreign_table`) is not allowed to be used and will raise an exception
```

Comment:
looks like one `:` is missing here after php

Reply:
Thanks, fixed it.